doc/source/getting_started/dpf_server.rst

```diff
@@ -120,3 +120,20 @@ DPF Server can be run in a Docker container.
     docker build . -t dpf-core:v2025.1.pre0 --build-arg DPF_VERSION=251
 
 5. To run the DPF Docker container, license it. For more information, see :ref:`DPF Preview License Agreement<target_to_license_terms>`.
+
+.. _ref_dpf_server_secure_mode:
+
+Run DPF Server in Secure mode wih mTLS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting with 26.1, DPF Server gRPC connections defaults to mTLS Transport. 
```

Comment:
we can also give the server version (11.0)?

Reply:
By the way this is also true for 252, 251, 242 and 241 with the latest service packs.
I'll mention that.